core/src/main/scala/org/apache/spark/status/storeTypes.scala

```diff
@@ -141,6 +141,7 @@ private[spark] object TaskIndexNames {
   final val STATUS = "sta"
   final val TASK_INDEX = "idx"
   final val COMPLETION_TIME = "ct"
+  final val PARTITION_ID = "pid"
```

Comment:
nit: "pid" seems to be more in terms of process id, may be possible to change that?